hzpzp/cards/hzpzp/hzpzp_MZp10000_MChi1/hzpzp_MZp10000_MChi1_proc_card.dat

```diff
@@ -0,0 +1,4 @@
+import model Higgs_hzpzp_MZp10000_MChi1
```

Comment:
hello, I assume you don't want to make the model name change when the Z'mass and chi mass changes?

+import model Higgs_hzpzp_MZp10000_MChi1